lib/nm.ts

```diff
@@ -0,0 +1,218 @@
+import * as dbus from 'dbus-as-promised';
+import * as os from 'os';
+import { Options } from './index';
+
+type ActiveConnection = string;
+type Connection = ActiveConnection;
+
+class NetworkManager {
+	private wiredReference:
```

Comment:
```suggestion
	private wiredReference?:
```
and drop the `undefined` below. With the two other instances below.